src/BuildScriptGenerator/Node/NodeScriptGeneratorOptionsSetup.cs

```diff
@@ -15,7 +15,7 @@ internal class NodeScriptGeneratorOptionsSetup : IConfigureOptions<NodeScriptGen
         internal const string NodeSupportedVersionsEnvVariable = "NODE_SUPPORTED_VERSIONS";
         internal const string NpmSupportedVersionsEnvVariable = "NPM_SUPPORTED_VERSIONS";
         internal const string LegacyZipNodeModules = "ENABLE_NODE_MODULES_ZIP";
-        internal const string NodeLtsVersion = "8.11.2";
+        internal const string NodeLtsVersion = NodeVersions.Node10Version;
```

Comment:
should this also move to the constants file?

Reply:
Makes sense